documentation/doc-Migration_Toolkit_for_Virtualization/modules/using-must-gather.adoc

```diff
@@ -0,0 +1,53 @@
+// Module included in the following assemblies:
+//
+// * documentation/doc-Migration_Toolkit_for_Virtualization/master.adoc
+
+[id='using-must-gather_{context}']
+= Using must-gather
+
+The `must-gather` tool for {project-first} collects logs, metrics, and information about {project-short} custom resources.
+
+.Prerequisites
+
+* You must be logged in to the {virt} cluster as a user with the `cluster-admin` role.
+* You must have the link:https://docs.openshift.com/container-platform/{ocp-version}/cli_reference/openshift_cli/getting-started-cli.html[{ocp} CLI (`oc`)] installed.
+
+.Procedure
+
+. Navigate to the directory where you want to store the `must-gather` data.
+. Run the `oc adm must-gather` command with the `--image` argument:
++
+[source,terminal]
+----
+$ oc adm must-gather --image=quay.io/konveyor/forklift-must-gather:latest
```

Comment:
The image name will be different downstream: `registry.redhat.io/rhmtv/rhmtv-must-gather:v2.0.0`
So, we probably need a variable here.

Reply:
Good idea